docs-site/src/markdown-pages/index.md

```diff
@@ -0,0 +1,113 @@
+architecture will be built to deliver that outcome. With future outcomes
+building and evolving, the architecture to support the new outcomes.
+
+_Multi Tenanted_
+
+From the ground up EOP is being built to support multiple councils. This will
+enable the effort to deliver an outcome by one council to be delivered to all
+councils for an incremental cost. This means in general, EOP will be agnostic of
+council internal systems with each council providing data to EOP in a
+standardised format. However, for some common systems used in a majority of
+councils, EOP can provide standardised adapters to allow the data to be
+
+_Modern Development Practices_
+
+We are using modern deliver practices ... why do you care? It supports long term
+maintainability and reduces the risk of the platform becoming legacy.
+
+## Data Sets
+
+TODO—List of data sets
+
+## Governance
+
+Being a sector wide initiative, there is a need for governance to support equity
+amongst the stakeholders. There are two groups for governing EOP. The first is a
+Steering Group on the strategic direction of EOP. The second is a Technical
+Working Group which will focus on the technical direction of EOP in terms of
+architecture, collaboration, development practices, and technical priorities.
+
+### Steering Group
+
+The steering group has a Terms of Reference formed which describes the
+responsibilities of the Steering Group as follows:
+
+- To set strategic level priorities and direction for the EOP Delivery Team
+  around which outcomes to focus on. The EOP Delivery Team sets technical
+  direction and operational priorities towards delivering on the strategic
+  priorities and direction set by the Steering Group.
+- To facilitate the access and management of funds to invest in, develop and
+  maintain EOP across the sector.
+- To facilitate the communication and progress of EOP to other areas that
+  Steering Group Members are included in – such as DSG, EDSIG, etc. along with
+  documenting feedback and providing back to other Steering Group Members and
+  EOP Delivery Team.
+- To receive and consider regular updates/reports then make recommendations
+  and/or endorsements as appropriate.
+- To identify where any strategic changes in the sector could impact delivery of
+  EOP, and vice versa.
+- To provide guidance on issues or risks that are escalated from the EOP
+  Delivery Team Members.
+- To resolve any dispute or disagreement that is raised through the governance
+  structure.
+
+### Technical Working Group
+
+A technical group is going to be setup in 2023 led by the EOP Delivery team.
+This will be the forum to share and collaborate on the technical solution across
+contributing councils and potential future contributors.
+
+- Setting the technical architecture for EOP
+- Setting the technical development practices for EOP
+- Setting the technical collaboration practices for EOP
+- Setting technical priorities for the EOP Delivery Team
```

Comment:
Is "Setting technical priorities for the EOP Delivery Team" too hands on for this group? My take is this should be decided by the team, with input from this group, alongside the Steerco and PO/Roadmap. Also, a little word-smithing of the bullet points:

>We imagine the activities of this group to be:
>- Discussion and feedback on major architectural concerns and decisions
>- Helping establish development practices for technical collaboration on EOP across councils
>- Feeding back on current technical priorities of the EOP Delivery Team
>- Bringing a cross-council perspective to technical development by sharing knowledge, challenges and opportunities from different councils